aws_topology/stackstate_checks/aws_topology/resources/lambda_event_source_mapping.py

```diff
@@ -0,0 +1,20 @@
+from ..utils import make_valid_data
+from .registry import RegisteredResource
+
+
+class lambnda_event_source_mapping(RegisteredResource):
```

Comment:
Class names should use CamelCase convention. Fix typo it should be `lambda_event_source_mapping`

Reply:
Good catch, doesn't flake8 allow for enforcing these CamelCase things?